### DISCUSSION TITLE
Add javascript to canvas.

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#4764] [Q&A] [answered]
Hi Arthur and other contributors!

Is there a way to set a javascript code to the Canvas and grapesjs save it in the final html export?

Just for context, i need to create some animations merging animate.css with wow.js

**Comment by rajevar** [comment, marked as answer]
this would add the js to the canvas. I dont think it will get added to the final html export, you might have to add by code.

`editor.Canvas.getDocument().body.insertAdjacentHTML(
			'afterend',
			"<script src='myscripts.js'></script>"
		);`
